src/app/components/your-work/your-work.component.html

```diff
@@ -1 +1,49 @@
+<ng-template #tableTemplatePartial let-row>
+  <td class="govuk-table__cell">{{ row.case_number }}</td>
+  <td class="govuk-table__cell">{{ row.courthouse_name }}</td>
+  <td class="govuk-table__cell">{{ row.hearing_date | date: 'dd MMM yyyy' }}</td>
+  <td class="govuk-table__cell">{{ row.transcription_type }}</td>
+  <td class="govuk-table__cell">{{ row.requested_ts | date: 'dd MMM yyyy HH:mm' }}</td>
+  <td class="govuk-table__cell">
+    <span [ngClass]="row.urgency === 'Overnight' ? 'moj-badge moj-badge--red' : ''">{{ row.urgency }}</span>
+  </td>
+  <td class="govuk-table__cell">
+    <a href="#" (click)="onViewWorkRequest($event, row)" class="govuk-link">View</a>
+  </td>
+</ng-template>
+
 <h1 class="govuk-heading-l">Your work</h1>
+<ng-container *ngIf="requesterRequests$ | async as requests; else loading">
+  <app-tabs>
+    <div *tab="'To do'; count: requests.todoRequests.length">
+      <ng-container *ngIf="requests.todoRequests.length">
+        <h2 class="govuk-heading-m">To do</h2>
+        <app-data-table id="todoTable" [rows]="requests.todoRequests" [columns]="readyColumns" [pagination]="false">
```

Comment:
Can you add `noDataMessage="There are no outstanding transcript requests"`

Reply:
Fixed